tools/pkglint/lib/rules.ts

```diff
@@ -1463,6 +1463,7 @@ export class UbergenPackageVisibility extends ValidationRule {
 
   // These include dependencies of the CDK CLI (aws-cdk).
   private readonly publicPackages = [
+    '@aws-cdk/cfnspec',
```

Comment:
This will also need to be backported onto the `master` branch. Just reduces the possibility of merge conflicts.

Reply:
https://github.com/aws/aws-cdk/pull/11552